modelstore/model_store.py

```diff
@@ -135,6 +135,8 @@ def list_models(self, domain: str, state_name: Optional[str] = None) -> list:
     """
     MODEL STATES: a model state is a string that has a 1:many relationship
     with models.
+
+    @TODO: There is no function to get the meta-data for a state
```

Comment:
I've added this in as a note for the future - we have `list_model_states()`, but no way to `get_model_state(state_name: str)` and I thought that might be useful to have